generators/new_collateral/initializer/generator.go

```diff
@@ -0,0 +1,70 @@
+package initializer
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dave/jennifer/jen"
+	"github.com/makerdao/vdb-mcd-transformers/generators/new_collateral/types"
+)
+
+type IGenerate interface {
+	GenerateFlipInitializer() error
+	GenerateMedianInitializer() error
+}
+
+type Generator struct {
+	Collateral                types.Collateral
+	MedianInitializerRequired bool
+}
+
+var initializerFileName = "initializer.go"
+
+func (g *Generator) GenerateFlipInitializer() error {
```

Comment:
Do the methods in this file require pointer receivers?

Reply:
good call!